.gitignore

```diff
@@ -12,3 +12,4 @@ bower_components/
 testBundle.js
 docs/build
 docgenInfo.json
+.idea/
```

Comment:
Use [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for the `.idea/`directory.  This way not all projects have to have this entry, also not all developers will use a JetBrains product so this ignore only applies to a subset of users.